test/classes/assignments/noAssignClass.chpl

```diff
@@ -42,55 +42,55 @@ class K {
   var x: int;
 }
 
-proc =(lhs: A, rhs: A) {
+operator =(lhs: A, rhs: A) {
```

Comment:
I thought I had a version of this test lying around with all of them copied, because making them into operator methods exposed a bug with the error message that I fixed in another branch.  Lemme go track that back down

Reply:
Yup!  I already extended this test to include a copy of those types but that specifically used operator methods instead.